canonical-websites/blog.ubuntu.com
# Feature/rename to blog

## Done

Updates logo and open graph thumbnail

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8023/](http://0.0.0.0:8023/)
- Check logo says blog and not insights
- Look at a post without a thumbnail and in the source make sure the open graph meta tag image says blog and not insights
